zipkin-server/storage-cassandra/src/main/java/zipkin/server/storage/cassandra/CassandraTableInstaller.java

```diff
@@ -43,6 +46,17 @@ public void start() {
 
   @Override
   public void whenCreating(Model model) throws StorageException {
+    if (ZipkinSpanRecord.INDEX_NAME.equals(model.getName())) {
+      // remove unnecessary columns
+      for (int i = model.getColumns().size() - 1; i >= 0; i--) {
+        final String columnName = model.getColumns().get(i).getColumnName().getStorageName();
+        if (StorageData.TIME_BUCKET.equals(columnName) ||
+            ZipkinSpanRecord.TIMESTAMP_MILLIS.equals(columnName) ||
+            JDBCTableInstaller.TABLE_COLUMN.equals(columnName)) {
```

Comment:
CassandraTableInstaller is referring to the JDBC implementation? Where does this TABLE COLUMN come from?

Reply:
> CassandraTableInstaller is referring to the JDBC implementation?

Nope, For now, the table installer only caches the table metadata. 

> Where does this TABLE COLUMN come from?

There are definitions in skywalking core that are not necessary for the query.